collector/statsdEmitter_test.go

```diff
@@ -0,0 +1,60 @@
+// +build unit
+
+// Copyright 2016 Mesosphere, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collector
+
+import (
+	"fmt"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestToStatsdLabel(t *testing.T) {
+	Convey("When converting a StatsEvent to a StatsD label", t, func() {
+
+		// Spot check a few event types
+		Convey("Known events should produce valid labels", func() {
+			So(toStatsdLabel(
+				StatsEvent{evttype: 2}), // 2 => TCPResolveFailed
```

Comment:
Let's not halt work due to this, we can come back and update the test once @nickbp is back in the office.